mkosi/qemu.py

```diff
@@ -144,19 +144,22 @@ def identify(cls, config: Config, path: Path) -> "KernelType":
             logging.warning("bootctl doesn't know kernel-identify verb, assuming 'unknown' kernel type")
             return KernelType.unknown
 
-        type = run(
-            ["bootctl", "kernel-identify", path],
-            stdout=subprocess.PIPE,
-            sandbox=config.sandbox(options=["--ro-bind", path, path]),
-        ).stdout.strip()
+        try:
+            type = run(
+                ["bootctl", "kernel-identify", path],
+                stdout=subprocess.PIPE,
+                sandbox=config.sandbox(options=["--ro-bind", path, path]),
+            ).stdout.strip()
+        except Exception:
```

Comment:
No blanket `except Exception`, please. You probably want
```suggestion
        except subprocess. CalledProcessError:
```
or even better
```suggestion
        except subprocess. CalledProcessError as e:
```